packages/perspective-viewer-datagrid/src/less/regular_table.less

```diff
@@ -120,8 +123,11 @@ td:focus {
     outline-width: 1px;
 }
 
-perspective-viewer.dragging regular-table {
-    pointer-events: none;
+perspective-viewer.dragging,
+:host-context(perspective-viewer.dragging) {
```

Comment:
I don't know how to test this rule - couldn't find anywhere where this rule is set, and various drag/drop operations I tried didn't seem to add the class name to the viewer.